rootfs/rootfs/etc/rc.d/xedaemon

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if [ -x /etc/init.d/xe-linux-distribution ]; then
+	/etc/init.d/xe-linux-distribution start
```

Comment:
Thanks! It will not start up if we're not on a Xen platform. 
1. /etc/init.d/xe-linux-distribution will start /usr/sbin/xe-daemon. [xe-linux-distribution.init#L42](https://github.com/xenserver/xe-guest-utilities/blob/boot2docker/mk/xe-linux-distribution.init#L42)
2. Then xe-daemon will quit if the VM is not run on a Xen platform. [xe-daemon.go#L48](https://github.com/xenserver/xe-guest-utilities/blob/boot2docker/xe-daemon/xe-daemon.go#L48)